03.bowling/bowling.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
```

Comment:
私なら map を使って書きますねー
また、Ruby では each で配列に詰め直す処理は、map で書く方が多いように感じます

例えば以下のようにするとか
```suggestion
shots = scores.map { |s| (s == "X") ? [10, 0] : s.to_i }.flatten
```

Reply:
とってもシンプルになりました!!こちらの書き方覚えておきます。
これからeachを使うときには「mapは使えないかな?」をあわせて検討するようにします。